src/interfaces.ts

```diff
@@ -14,7 +14,7 @@ export enum ElectronVersionSource {
   remote = 'remote',
   local = 'local'
 }
-export interface NpmVersion {
+export interface Version {
```

Comment:
Renamed from `NpmVersion` to `Version` here because it's no longer from NPM. This causes a lot of things to be renamed.